addons/mail/static/src/owl/components/composer_text_input/composer_text_input.js

```diff
@@ -31,530 +20,95 @@ class ComposerTextInput extends Component {
-
-        return { $textarea, editable };
-    }
-
-    /**
-     * Configure tribute with the contenteditable made with summernote editor.
-     * This enables mentions.
-     *
-     * @private
-     * @param {Object} param0
-     * @param {HTMLElement} param0.editable
-     * @return {Object} tribute object
-     */
-    _configTribute({ editable }) {
-        const tribute = new window.Tribute({
-            collection: [
-                this._configTributeCollectionItemCannedResponse(),
-                this._configTributeCollectionItemChannel(),
-                this._configTributeCollectionItemCommand(),
-                this._configTributeCollectionItemPartner(),
-            ],
-        });
-
-        tribute.attach(editable);
-        return tribute;
-    }
-
-    /**
-     * Configure canned responses with tribute.
-     *
-     * @private
-     * @return {Object}
-     */
-    _configTributeCollectionItemCannedResponse() {
-        const self = this;
-        const collectionItem = {
-            lookup: 'source',
-            menuItemTemplate(item) {
-                return self.env.qweb.renderToString('mail.component.ComposerTextInput.CannedReponseMentionMenuItem', {
-                    isMobile: self.storeProps.isMobile,
-                    item,
-                });
-            },
-            selectTemplate(item) {
-                return item ? item.original.substitution : null;
-            },
-            trigger: ':',
-            values(keyword, callback) {
-                const cannedResponses = self._searchCannedResponseSuggestions(keyword);
-                callback(cannedResponses);
-            },
-        };
-        return collectionItem;
-    }
-
-    /**
-     * Configure channel mentions with tribute.
-     *
-     * @private
-     * @return {Object}
-     */
-    _configTributeCollectionItemChannel() {
-        const self = this;
-        const collectionItem = {
-            lookup: 'name',
-            menuItemTemplate(item) {
-                return self.env.qweb.renderToString('mail.component.ComposerTextInput.ChannelMentionMenuItem', {
-                    isMobile: self.storeProps.isMobile,
-                    item,
-                });
-            },
-            selectTemplate(item) {
-                if (!item) {
-                    // no match keeps mentioning state, hence handle no item selection
-                    return null;
-                }
-                return self.env.qweb.renderToString('mail.component.ComposerTextInput.ChannelMentionSelectItem', { item });
-            },
-            trigger: '#',
-            values(keyword, callback) {
-                self._searchChannelMentionSuggestions(keyword, channels => callback(channels));
-            },
-        };
-        return collectionItem;
-    }
-
-    /**
-     * Configure commands with tribute.
-     *
-     * @private
-     * @return {Object}
-     */
-    _configTributeCollectionItemCommand() {
```

Comment:
If you remove all usages of tribute, you can remove the lib files/import too.

(revert commit [IMP] mail: import tribute)

Reply:
Done